### PR TITLE
Fixes for indexer show searching

### LINKED ISSUE #3720: Searcing with danish characters
Unable to perform searches with æ ø å. oe instead of ø does not work neither.

Branch/Commit: master/bd0832551d836713481dd0a93f4118372bb5d963
OS: Windows Server 2012
What you did: Searching for show: Børnelokkerne
What happened: No results
What you expected: Expected to find this show: https://www.thetvdb.com/?tab=series&id=316315&lid=10
Logs:
2018-02-06 23:48:04 INFO     Thread-37 :: [bd08325] Error searching for show: Show search failed in getting a result with reason: Show bÃƒÆ’Ã‚Â£Ãƒâ€šÃ‚Â¸rnelokkerne not found
2018-02-06 23:48:04 INFO     Thread-37 :: [bd08325] Error searching for show: Show search failed in getting a result with reason: HTTPError

### LINKED ISSUE #4246: [API v1] sb.searchindexers does not return anything
Hi guys,

I received a bug report for my Android app (https://github.com/MGaetan89/ShowsRage/issues/149) related to the search feature in the API v1.
When using this entry point, Medusa (version 0.2.4) does not return anything, while the web UI returns some shows for the same query.
Here is a sample request that returns nothing for me from the API (in the API builder), and a lot of shows on the web UI:
`/api/v1/{api_key}/?cmd=sb.searchindexers&name=how`

Could you have a look to see what is going on?
I know that you have an API v2 now, and I am planning to include it at some point. But until then, I will need v1 to work as expected ;)

### LINKED ISSUE #4248: Edit show stops/hangs with "error 500"
V 0.24, unclear what triggers this. Usually re-starting helps to recover.
Screenshot:
http://www.imagebam.com/image/866b7f872960494
Edit: Test  clip (replaced):
https://drive.google.com/file/d/1cLlRIYILPpmhGKm-skV-qrpzLp-RYCO1/view?usp=sharing

Seems to also be  confirmed inn Slack..

### PR DESCRIPTION
**Pull request by sharkykh**
This PR combines #4251 and #4255.

----

- `tvdbapiv2` lib updated (see https://github.com/pymedusa/tvdbv2/pull/2 for a more details)
    Fixes #4248 - In short - the response from the API wrapper wasn't unicode.
	Steps to reproduce the issue:
	- Add the show `The Crossing` from TVDB (id: 328631)
	- After the show is fully loaded, restart Medusa. Try and edit the show - works.
	- "Force Full Update" of the show.
	- After the show is fully updated, try and edit the show - error.
	  With the proposed fix - it works.
- Fixes #4246 - No search results in APIv1 sb.searchindexers 
	Also fixes #1895 (#1890) - tuple index out of range (when searching tvmaze, tmdb or all indexers)
- Fix searching indexers for shows in other languages.
	Fixes #3720

Tested using the new show form, API v1 command, and the edit show form.
Also tested the `tvdbapiv2` library separately.

### CODE REVIEW

File: lib/tvdbapiv2/__init__.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # coding=utf-8
 
-from __future__ import absolute_import
+from __future__ import absolute_import, unicode_literals
```
[the remark this answers is not in the document]
Comment (sharkykh) [reply lib/tvdbapiv2/__init__.py]: @p0psicles 
You're talking about #1448.
`requests` was using a vendored version of pyOpenSSL,
and installing a local version of pyOpenSSL caused the issue.
See https://github.com/requests/requests/issues/3824

`requests` no longer uses any vendored packages, so it makes sense that this issue is fixed.
I installed `requests[security]` (installs pyOpenSSL) and I can't replicate it on v2.18.4 (`ext/` folder has the same version).

[the remark this answers is not in the document]
Comment (p0psicles) [reply lib/tvdbapiv2/__init__.py]: @ratoaq2 do you remember why we reverted back then?

[the remark this answers is not in the document]
Comment (medariox) [reply lib/tvdbapiv2/__init__.py]: I don't remember what the problem was at the time tbh.

[the remark this answers is not in the document]
Comment (p0psicles) [reply lib/tvdbapiv2/__init__.py]: Lets merge and just keep an eye out. Rest of pr LGTM

**Pull request merged**